packages/get-context/src/index.ts

```diff
@@ -290,9 +290,17 @@ async function purgeModulesDirsOfImporter (
   }
 }
 
+// A RegExp used to match files that should be kept,
+// matches all .dot_files excluding the .bin directory and pnpm's files:
+const KEEP_FILES = /^\.(?!(bin|modules\.yaml|pnpm)$)/;
```

Comment:
seems like a regex is not needed, just an array of files

```
const KNOWN_DOT_FILES = new Set([
  '.bin',
  '.modules.yaml',
  '.pnpm',
])
```

although, the `.pnpm` directory may be customized via the virtual-store-dir setting